### PR TITLE
chore: use package for common_ui

### PR DESCRIPTION
**Pull request by MaYohann**
Use registry package in local & ci
Could you install it on your local env pls? @ismailsunni @toni-garcia-camptocamp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,37 @@
...
+
+Then, in your personnal ~/.npmrc you need to add this line:
+
+```shell
+//npm.pkg.github.com/:_authToken=<YOUR PAT>
+```
+
+Or, in the project ./npmrc file, you can replace NPM_TOKEN by your PAT.
+
+**Careful!** If you change the project npmrc file, you must be careful and not commit the change. The docker image in the CI is build by replacing the ${NPM_TOKEN} by the GITHUB_TOKEN of the repository.
```
Comment (ismailsunni) [inline README.md:22]: Should we put it on `.gitignore`? We can create a sample file and put some instructions for the user to copy and edit the file.

Comment (ismailsunni) [reply README.md:22]: Ah I see, so I can put the PAT on the project .npmrc or in `~/.npmrc`

File: README.md [hunk 2]
```diff
@@ -2,6 +2,37 @@
 
 ## Quick start
 
+### Package common_ui from github registry
+
+#### Installation
+
+This app install the [@sigrennesmetropole/cooperation_jn_common_ui](https://github.com/sigrennesmetropole/cooperation_jn_common_ui/pkgs/npm/cooperation_jn_common_ui) package.
+Although this package is public, you still need a Private Access Token (PAT) from github to download it.
```
Comment (ismailsunni) [inline README.md:10]: Do you know why we need this PAT? It's strange to have this kind of thing while all is public :)

**Review by ismailsunni — COMMENTED** [review]
Code is LGTM, I will try it

**Review by toni-garcia-camptocamp — COMMENTED** [review]
LGTM and the installation on my local with your instruction work well

**Pull request merged**